tools/generate_github_changelog.py

```diff
@@ -0,0 +1,4 @@
+import subprocess
```

Comment:
```suggestion
"""
Script to render to terminal, the changelog entry for the latest version in markdown so it can be copied and pasted
for a GitHub release.

It requires pandoc to be installed on your system.
"""

import subprocess
```